modules/api/src/main/java/org/eclipse/xpanse/api/MonitorApi.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ * SPDX-FileCopyrightText: Huawei Inc.
+ *
+ */
+
+package org.eclipse.xpanse.api;
+
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.Parameter;
+import io.swagger.v3.oas.annotations.tags.Tag;
+import jakarta.annotation.Resource;
+import java.util.List;
+import java.util.UUID;
+import lombok.extern.slf4j.Slf4j;
+import org.eclipse.xpanse.modules.monitor.Metric;
+import org.eclipse.xpanse.orchestrator.OrchestratorService;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.web.bind.annotation.CrossOrigin;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestController;
+
+/**
+ * Monitor Api.
+ */
+@Slf4j
+@CrossOrigin
+@RestController
+@RequestMapping("/xpanse")
+public class MonitorApi {
+
+    @Resource
+    private OrchestratorService orchestratorService;
+
+    /**
+     * Get Monitor Metric.
+     *
+     * @param id Service ID.
+     */
+    @Tag(name = "Monitor",
+            description = "APIs to Monitor Metric.")
```

Comment:
APIs to Monitor Metric ->please use either of the two as description.

1. APIs to get metrics of deployed services. 
2. APIs to monitor deployed services.